mgmt_cli_test.py

```diff
@@ -1550,3 +1551,88 @@ def test_restore_data_without_manager(self):
                                "Please provide the 'mgmt_reuse_backup_snapshot_name' parameter.")
 
         self.test_restore_from_precreated_backup(snapshot_name, restore_outside_manager=True)
+
+
+class ManagerBackupRestoreConcurrentTests(ManagerTestFunctionsMixIn):
+    def test_backup_benchmark(self):
+        def report_to_argus(data, label):
+            send_manager_benchmark_results_to_argus(
+                argus_client=self.test_config.argus_client(),
+                result=data,
+                sut_timestamp=mgmt.get_scylla_manager_tool(
+                    manager_node=self.monitors.nodes[0]).sctool.client_version_timestamp,
+                row_name=label,
+            )
+
+        def create_backup(label):
+            mgr_cluster = self.ensure_and_get_cluster(
+                mgmt.get_scylla_manager_tool(manager_node=self.monitors.nodes[0]))
+
+            backup_start_time = time.time()
+            task = mgr_cluster.create_backup_task(location_list=self.locations, rate_limit_list=["0"])
+            backup_task_status = task.wait_for_uploading_stage(timeout=200000)
+            assert backup_task_status, "Backup has failed!"
+            backup_end_time = time.time()
+            InfoEvent(message=f'Backup without upload took: {backup_end_time - backup_start_time:.2f}s.').publish()
+            upload_start_time = time.time()
+            backup_status = task.wait_and_get_final_status(timeout=200000)
+            upload_end_time = time.time()
+            assert backup_status == TaskStatus.DONE, "Backup upload has failed!"
+            InfoEvent(
+                message=f'Backup upload upload took: {upload_end_time - upload_start_time}.').publish()
+            InfoEvent(
+                message=f'Backup total time is: {(backup_end_time - backup_start_time) + (upload_end_time - upload_start_time)}.').publish()
+            backup_report = {
+                "backup time": int(backup_end_time - backup_start_time),
+                "upload time": int(upload_end_time - upload_start_time),
+                "total": int((backup_end_time - backup_start_time) + (upload_end_time - upload_start_time)),
+            }
```

Comment:
You may consider an option to get the whole backup process time as `task.duration` property.
It will minimize the number of calculations here

Reply:
Done